.trivyignore

```diff
@@ -23,3 +23,6 @@ CVE-2025-22869
 
 ## CVE-2025-22874 (usr/bin/ollama, usr/local/bin/aws-sso, usr/local/bin/helm, usr/bin/kubectl)
 CVE-2025-22874
+
+## CVE-2025-22088 (kernel)
+CVE-2025-22088
```

Comment:
Just building this container again is enough to fix, don't need to add to Trivy's ignore.

My suggestion would be to patch this image